addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -213,6 +225,9 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
         }
 
         this.currentScreen = screenToDisplay;
+        if (!['question', 'nextQuestion'].includes(screenToDisplay)) {
```

Comment:
I don't think it can ever be "nextQuestion" when going back.
Also, shouldn't we update the tooltip when going back to the question? It might keep the "wrong" tooltip, right? (aka the one from the screen we are coming back from).

Reply:
This is the same case when the tooltip will be updated from the widget re-render.

Reply:
Then don't you need to add "previousQuestion" to the list to avoid the same flicker you mentioned ?